js/src/routes.js

```diff
@@ -60,6 +62,7 @@ const contractsRoutes = [
 ];
 
 const settingsRoutes = [
+  { path: 'advanced', component: SettingsAdvanced },
```

Comment:
100% agreed that we will need this going forward. My concern is adding more bulk right at the end of the release when there is no way it can actually go through proper testing where the actual feature has no real benefit apart from a small slice. 

So currently only enable in dev (`if (NODE_ENV !=== 'development) return null`) and keep the dropdown it in Parity so we can be consistent going forward.